core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -455,9 +460,10 @@ private void propagateRequestSideLog(RequestLogAccess child) {
              .thenAccept(log -> serializationFormat(log.scheme().serializationFormat()));
         child.whenAvailable(RequestLogProperty.NAME)
              .thenAccept(log -> {
+                 final String serviceName = log.serviceName();
                  final String name = log.name();
-                 if (name != null) {
-                     name(name);
+                 if (serviceName != null && name != null) {
```

Comment:
Shouldn't this only check `name`? `serviceName` shouldn't be required.

Reply:
Oops... Yes!